recipes-libretro/desmume-libretro/desmume-libretro_git.bb

```diff
@@ -16,8 +16,8 @@ LIBRETRO_PLATFORM_append_opengles = "-gles"
 LIBRETRO_GIT_REPO = "github.com/libretro/desmume.git"
 LIBRETRO_CORE = "desmume/src/frontend/libretro/desmume"
 
-# Unable to compile on 64-bit platforms
-COMPATIBLE_MACHINE = "(32bit)"
+EXTRA_OEMAKE_aarch64 += "DESMUME_JIT=0"
+EXTRA_OEMAKE_X86_64 += "DESMUME_JIT=0"
```

Comment:
From what I know mixing conditional overrides with `+=` operator is undefined behavior. Proper way is to use
`EXTRA_OEMAKE_append_aarch64 = " DESMUME_JIT=0"` and `X86_64` should be `x86-64`.

Overall with additional overrides provided indirectly by libretro class (retro-overrides). This can be expressed as 
`EXTRA_OEMAKE_append_64bit = " DESMUME_JIT=0"` this should apply for multiple 64bit architectures.

Reply:
You're right. Changed to EXTRA_OEMAKE_append_64bit

Reply:
> From what I know mixing conditional overrides with += operator is undefined behavior. 

I think that I was wrong about `+=` and undefined behavior stuff. `EXTRA_OEMAKE_64bit += "DESMUME_JIT=0"` should also work fine in this case.